pkgs/applications/audio/furnace/default.nix

```diff
@@ -67,7 +75,12 @@ stdenv.mkDerivation rec {
     "-DSYSTEM_RTMIDI=ON"
     "-DSYSTEM_SDL2=ON"
     "-DSYSTEM_ZLIB=ON"
+    "-DSYSTEM_PORTAUDIO=ON"
     "-DWITH_JACK=${if withJACK then "ON" else "OFF"}"
+    "-DWITH_PORTAUDIO=ON"
+    "-DWITH_RENDER_SDL=ON"
+    "-DWITH_RENDER_OPENGL=ON"
```

Comment:
Looks like this doesn't work on darwin?

Reply:
Will look into it tonight.

Reply:
From a quick look, might not be supported by upstream: https://github.com/tildearrow/furnace/blob/v0.6pre16/CMakeLists.txt#L70